server/utils/utils.test.ts

```diff
@@ -490,3 +491,20 @@ describe('calculateAge', () => {
     expect(calculateAge(twentyYearsAgo)).toBe(19)
   })
 })
+
+describe('generateCurrentYear', () => {
+  it('returns current year', () => {
+    const currentYear = moment().year()
+    expect(generateCurrentYear()).toBe(currentYear)
+  })
+
+  it('does not return previous year', () => {
+    const previousYear = moment().subtract(1, 'years').year()
+    expect(generateCurrentYear()).not.toBe(previousYear)
+  })
```

Comment:
When your function is always returning a (relatively) constant value it's not really worth testing whether its value is other things once you've confirmed what that value is. 

Injecting the concept of "now" might be better as this is just a form of double accounting - you're asserting the test behaves how you have implemented it.

A basic test might be 
expect(generateCurrentYear(2020-04-05 00:01:20)).toBe(2020) 

Other possible interesting test cases might include? 
Now is the very start of the year
Now is the end of the year

Reply:
(if you do inject the concept of now, you might want have a default param of something that provide now, as passing now in via nunjucks might be faffy)

Reply:
Cool, refactored this in the latest commit